src/app/job/[id]/page.tsx

```diff
@@ -0,0 +1,62 @@
+import { ReactElement } from "react";
+
+import Image from "next/image";
+
+import { notFound } from "next/navigation";
+
+import MingcuteBookmarkLine from "@/icons/MingcuteBookmarkLine";
+
+import { jobs } from "@/mock/jobs";
+
+import { JobModel } from "@/models/job.model";
+
+import styles from "./page.module.css";
+
+type Props = {
+  params: { id: string };
+};
+
+export default async function Page({ params }: Props): Promise<ReactElement> {
+  const job = await getJob(params.id);
+
+  if (!job) {
+    return notFound();
+  }
+  return (
+    <div className={styles.page}>
+      <header>
+        <div className={styles.company}>
+          <div className={styles["company-image"]}>
+            <Image src={job.image} alt="job image" width={24} height={24} />
```

Comment:
وقتی از alt استفاده میکنید، باید عکس رو توصیف کنید
اگه صرفاً جنبه‌ی تزئیتی داره، میتونید خالی بذاریدش

```suggestion
            <Image src={job.image} alt="" width={24} height={24} />
```

Reply:
logo company گذاشتم